airbyte-workers/src/main/java/io/airbyte/workers/DbtTransformationRunner.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.workers;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.airbyte.commons.io.IOs;
+import io.airbyte.commons.io.LineGobbler;
+import io.airbyte.commons.resources.MoreResources;
+import io.airbyte.config.OperatorDbt;
+import io.airbyte.workers.normalization.NormalizationRunner;
+import io.airbyte.workers.process.ProcessBuilderFactory;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.apache.tools.ant.types.Commandline;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DbtTransformationRunner implements AutoCloseable {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(DbtTransformationRunner.class);
+  private static final String DBT_ENTRYPOINT_SH = "entrypoint.sh";
```

Comment:
It'd probably be safer to use absolute paths so the entrypoint isn't dependent on the working directory.

Reply:
this constant is defining the filename so it drives how to name the script file.
But when it's invoked, it's called with the absolute path so there wouldn't be dependency with working directory.